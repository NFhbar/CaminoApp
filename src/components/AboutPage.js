import React from 'react';
import { NavLink } from 'react-router-dom';
import { row, column } from 'reactstrap';

const AboutPage = () => (
  <div className="row">
    <div className="col-sm-12">
      <p>From About</p>
    </div>
  </div>
);

export default AboutPage;
