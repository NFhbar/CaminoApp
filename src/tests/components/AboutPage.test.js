import React from 'react';
import { shallow } from 'enzyme';
import AboutPage from '../../components/AboutPage';

test('should render About Page correctly', () => {
  const wrapper = shallow(<AboutPage />);
  expect(wrapper).toMatchSnapshot();
});
