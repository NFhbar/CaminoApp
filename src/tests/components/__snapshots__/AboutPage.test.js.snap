// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render About Page correctly 1`] = `
<div
  className="row card-body"
>
  <div
    className="col-sm-12"
  >
    <div
      className="jumbotron"
    >
      <h1
        className="display-4"
      >
        React-Redux App
      </h1>
      <p
        className="lead"
      >
        This App is built by
        <a
          href="https://www.linkedin.com/in/nicolas-frega/"
        >
           Nicolas Frega. 
        </a>
      </p>
      <hr
        className="my-4"
      />
      <p>
        The App is built with React, Redux, Bootstrap, Firebase, and Chartjs. It is deployed using Heroku. It mimics some of the behavior of a store App in which the admin can add items to sell at a given price. Added items are randomly generated sold amounts for 4 months for displaying purposes. Items can be sorted by price or amount sold. The items are then rendered dynamically to the page.
      </p>
      <p
        className="lead"
      >
        <a
          className="btn btn-primary btn-lg"
          href="https://github.com/NFhbar/CaminoApp"
          role="button"
        >
          View github
        </a>
      </p>
    </div>
  </div>
</div>
`;
